### DISCUSSION TITLE
How to install fpv-video on Ubuntu?

### DISCUSSION THREAD

**Discussion by zhuwei** [svpcom/wfb-ng#297] [Questions] [answered]
Didn't see the relevant tutorial

**Comment by svpcom** [comment]
https://github.com/svpcom/wfb-ng/wiki/Setup-HOWTO

**Comment by zhuwei** [comment]
@svpcom Do you mean this?
gst-launch-1.0 videotestsrc pattern=black ! videoconvert ! \
   video/x-raw,format=NV12,framerate=30/1,width=1280,height=720 ! \
   x264enc bitrate=4000 tune=zerolatency ! rtph264pay config-interval=1 ! \
   udpsink host=127.0.0.1 port=5600

**Reply by svpcom** [reply, marked as answer]
Yes, this is example for test video source. If you want to use real camera then it depends from camera type. 
For logitech c920 and generic ip-camer see examples in: https://github.com/svpcom/wfb-ng/blob/master/scripts/systemd/fpv-camera.service

**Comment by zhuwei** [comment]
@svpcom thank you very much!!!
